android/recipes/src/main/java/kmp/android/recipes/navigation/RecipesDestination.kt

```diff
@@ -0,0 +1,34 @@
+package kmp.android.recipes.navigation
+
+import androidx.annotation.StringRes
+import kmp.android.shared.Feature
+
+sealed class RecipesDestination(
+    val route: String,
+    @StringRes val name: Int,
+    @StringRes val description: Int,
+) {
+    companion object {
+        val root: String = Feature.Recipes.route
+    }
+
+    object List : RecipesDestination("$root/list", 0, 0)
+
+    object Rope : RecipesDestination(
+        "$root/rope",
```

Comment:
Nit.: When there are more arguments, especially when they are each on a separate line, it's nice to name the arguments for better readability:
```
route = "$root/rope",
name = kmp.android.shared.R.string.recipes_view_rope_name,
description = kmp.android.shared.R.string.recipes_view_rope_description,
```